app/src/main/java/com/isc/hermes/controller/PopUp/PopUpWarningIncorrectData.java

```diff
@@ -0,0 +1,60 @@
+package com.isc.hermes.controller.PopUp;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Build;
+import android.view.Gravity;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import com.isc.hermes.MainActivity;
+import com.isc.hermes.R;
+import com.isc.hermes.database.AccountInfoManager;
+import com.isc.hermes.model.User.UserRepository;
+
+public class PopUpWarningIncorrectData extends PopUp{
```

Comment:
This class don't have JavaDocs, add JavaDocs.

Reply:
done